bridges/modules/grandpa/src/lib.rs

```diff
@@ -98,6 +98,7 @@ pub mod pallet {
 	#[pallet::config]
 	pub trait Config<I: 'static = ()>: frame_system::Config {
 		/// The overarching event type.
+		#[allow(deprecated)]
```

Comment:
Why not just to remove all `type RuntimeEvent:` now? Or am I missing anything?

Reply:
Would break a lot of code :) We could do this always when we change the `Config` any way.